sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -65,6 +65,7 @@
 from apache_beam.transforms import environments
 from apache_beam.transforms import userstate
 from apache_beam.transforms import window
+from apache_beam.transforms.deduplicate import DeduplictaionWithinDuration
```

Comment:
Generally we've been following the convention of importing modules rather than objects from modules. http://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Thanks for the pointer!